apps/admin/backend/src/store.ts

```diff
@@ -1501,6 +1501,74 @@ export class Store {
     }
   }
 
+  getCastVoteRecord({
```

Comment:
This mirrors the function above it, just returning a single CVR instead of all of them

Reply:
Thanks for the suggestion, addressed in [c92183d](https://github.com/votingworks/vxsuite/pull/5815/commits/c92183d08afbef741a5a0d0e4a8b3c875515e014) by removing `getCastVoteRecord` and instead added an optional `cvrId` filter to `getCastVoteRecords`